### DISCUSSION TITLE
用图数据库可视化探索 Chia Network 区块链数据 - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#5] [Comments]
# 用图数据库可视化探索 Chia Network 区块链数据 - siwei.io

Nebula Graph 生态中有哪些 Spark 项目？ 本文为大家介绍 Spark-connector（包括 PySpark）， Nebula Algorithm 和 Nebula Exchange。

https://siwei.io/nebula-chia/

**Comment by wey-gu** [comment]
这篇文章我主要着重做了链数据的分析和导入，之后我会利用图计算探索更多的 Chia Network 洞察。
